windows/vbs_enable_disable/vbs_enable_disable.yml

```diff
@@ -29,13 +29,17 @@
             skip_reason: "Not Applicable"
           when: vm_firmware | lower != 'efi'
 
-        - name: "Set ESXi CPU model"
-          ansible.builtin.set_fact:
-            esxi_cpu_model: |-
-              {%- if 'AMD' in esxi_cpu_model_info -%}AMD
-              {%- elif 'Intel' in esxi_cpu_model_info -%}Intel
-              {%- else -%}Others
-              {%- endif -%}
+        - name: "Skip test case"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: >-
+              Skip test case due to using Intel CPUs for VBS requires vSphere 6.7 or later and using AMD CPUs for VBS requires 
+              vSphere 7.0 Update 2 or later. Current CPU is {{ esxi_cpu_vendor }} and hardware version is {{ vm_hardware_version_num }}.
+              Please refer to https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-CE292D3F-D4AC-4607-B262-DE19CE6E9F6B.html
+            skip_reason: "Not Supported"
+          when: >-
+            (vm_hardware_version_num | int < 14 and esxi_cpu_vendor | lower == "intel") or
+            (vm_hardware_version_num | int < 19 and esxi_cpu_vendor | lower == "amd")
```

Comment:
`esxi_cpu_vendor`'s value is in lower case, so you can directly use it without `lower` filter. And I suggest you  use hardware version instead of vSphere version in skip message, because the limitation is coming from the hardware version actually. For example, if the vSphere version is 8.0.2 but VM's hardware version is 13, it should not work for VBS.

Reply:
Sure, thanks.